public/v4/apps/invoiceplane.yml

```diff
@@ -0,0 +1,67 @@
+captainVersion: 4
+
+services:
+    $$cap_appname-db:
+        image: woahbase/alpine-mysql:x86_64
```

Comment:
any reason not to use the official mysql image?

Reply:
The official MySQL image is 145 MB, alpine-mysql is 70 MB. 
But we can use teh official one if you prefer.
Concerning the conflicts, I do not understand, because I use prettier and with prettier the string are enclosed in double quotes (") automatically when I format the YML, not with single quotes (').

Reply:
> The official MySQL image is 145 MB, alpine-mysql is 70 MB.
> But we can use the official one if you prefer.
Yea I think it's better to go with the official version as this version doesn't even allow the users to set the version of MySQL engine. The tags only include the architecture. 

> Concerning the conflicts, I do not understand, because I use prettier and with prettier the string are enclosed in double quotes (") automatically when I format the YML, not with single quotes (').

You just need to back merge master into your branch, or you can rebase. Both work.
If it complicates things for you, you can go with the ultimate option:
- Back up your template
- Delete your forked repository
- Fork again and create the PR

Reply:
And I have just discovered that there is the option "--single-quote" in prettier that I was not using !
I'll try to resolve teh conflicts. Thanks.

Reply:
As for prettier, you can just simply run
```
npm i
npm run formatter-write
```